extensions/amp-wpm-player/validator-amp-wpm-player.protoascii

```diff
@@ -0,0 +1,135 @@
+#
+# Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-wpm-player
+  html_format: AMP
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-wpm-player"
+    version: "0.1"
+    version: "latest"
+  }
+  attr_lists: "common-extension-attrs"
+}
+tags: {  # <amp-wpm-player>
+  html_format: AMP
+  tag_name: "AMP-WPM-PLAYER"
+  requires_extension: "amp-wpm-player"
+  attrs: {
+    name: "url"
+    value_url: {
+      protocol: "https"
+      allow_relative: false
+      allow_empty: false
+    }
+    mandatory_oneof: "['url', 'clip']"
+  }
+  attrs: {
+    name: "clip"
+    mandatory_oneof: "['url', 'clip']"
+  }
+  attrs: {
+    name: "adv"
+    value_casei: "true"
+    value_casei: "false"
+  }
+  attrs: {
+    name: "title"
+  }
+  attrs: {
+    name: "screenshot"
+    value_url: {
+      protocol: "https"
+      allow_relative: false
+      allow_empty: false
+    }
+  }
+  attrs: {
+    name: "forcerelated"
+    value_casei: "true"
+    value_casei: "false"
+  }
+  attrs: {
+    name: "hiderelated"
+    value_casei: "true"
+    value_casei: "false"
+  }
+  attrs: {
+    name: "hideendscreen"
+    value_casei: "true"
+    value_casei: "false"
+  }
+  attrs: {
+    name: "mediaEmbed"
+  }
+  attrs: {
+    name: "extendedrelated"
+    value_casei: "true"
+    value_casei: "false"
+  }
+  attrs: {
+    name: "skin"
+  }
+  attrs: {
+    name: "showlogo"
+    value_casei: "true"
+    value_casei: "false"
+  }
+    attrs: {
+    name: "watermark"
+    value_casei: "true"
+    value_casei: "false"
+  }
+  attrs: {
+    name: "qoeEventsConfig"
+  }
+  attrs: {
+    name: "advVastDuration"
+  }
+  attrs: {
+    name: "vastTag"
```

Comment:
If this is an URL, as per the markdown file, perhaps validate with a `value_url`.